components/docker-auth/cmd/authn/authentication.go

```diff
@@ -148,16 +164,16 @@ func isJWT() bool {
 	isJWTEnv := os.Getenv("IS_JWT")
 	var isJWT bool
 	if len(isJWTEnv) == 0 {
-		glog.Error("Error: IS_JWT environment variable is empty")
-		extension.Exit(extension.ErrorExitCode)
+		log.Println("Error: IS_JWT environment variable is empty")
+		os.Exit(extension.ErrorExitCode)
 	} else {
 		if isJWTEnv == "true" {
```

Comment:
Use case insensitive comparison

Reply:
The jwt validating implementation is removed from the current implementation.